tsfc/driver.py

```diff
@@ -377,7 +377,11 @@ def compile_expression_dual_evaluation(expression, to_element, *,
             quad_rule = QuadratureRule(point_set, to_element._weights)
             config["quadrature_rule"] = quad_rule
 
+        # evaluate expression at the points specified by point_set
         expr, = fem.compile_ufl(expression, **config, point_sum=False)
+        # the point set free indices should now be free indices of the complied
+        # expression
+        assert all(i in expr.free_indices for i in point_set.indices)
```

Comment:
I think this is:
```
set(point_set.indices) <= set(expr.free_indices)
```
?
Which is clearer, I think.

Reply:
Changed in d711e722d19ce86ae5fca8b60e01492bd55e6ae6